<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string comment="Application Help tooltip"/>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <widget class="QLineEdit" name="lineEdit_fichier_source">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Spécifier un fichier source.</string>
   </property>
   <property name="locale">
    <locale language="French" country="France"/>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_browse_source_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Ouvrir le selecteur de fichier</string>
   </property>
   <property name="locale">
    <locale language="French" country="France"/>
   </property>
   <property name="text">
    <string>Sélecteur du fichier source Base BQ Sqlite</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titre_application">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Outil pour ...</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="locale">
    <locale language="French" country="France"/>
   </property>
   <property name="text">
    <string>Diffunews : Créateur de newletters</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fichier_destination">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Spécifier un Fichier de destination.</string>
   </property>
  </widget>
  <widget class="Line" name="decorslinebas">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_action">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Lance la génération du fichier .cvs">Action de</string>
   </property>
   <property name="locale">
    <locale language="French" country="France"/>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_browse_destination_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Ouvrir le selecteur de fichier</string>
   </property>
   <property name="locale">
    <locale language="French" country="France"/>
   </property>
   <property name="text">
    <string>Sélecteur du fichier destination pdf</string>
   </property>
  </widget>
  <widget class="Line" name="decorslinehaut">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_zone_resultataction">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>710</y>
     <width>541</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Résultat">Indique le resultat  du traitement.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_exit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>710</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Quitter l'application">Quitter l'application.</string>
   </property>
   <property name="locale">
    <locale language="French" country="France"/>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_aide">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Ouvre la fenêtre d'aide détaillée.</string>
   </property>
   <property name="statusTip">
    <string extracomment="Application Help "/>
   </property>
   <property name="locale">
    <locale language="French" country="France"/>
   </property>
   <property name="text">
    <string>Aide</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_location">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choisir un emplacement de la bibliothèque.</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_location">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Localisation des livres dans la bibliothèque :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_originality">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choisir un emplacement de la bibliothèque.</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_originality">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>L'origine des livres :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_targetaudience">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choisir un emplacement de la bibliothèque.</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_originality_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Le public concerné :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_requete_sql">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>831</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_creation_requete">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>580</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Creer requete SQL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nbretotal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre total de documents dans la base :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbretotal">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbretotal_filtre_location">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nbretotal_apresfiltre_location">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de documents dans la base après filtrage localisation :</string>
   </property>
  </widget>
  <widget class="Line" name="decorslinebas_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="decorslinebas_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbretotal_filtre_originality">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nbretotal_apres_filtre_originality">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de documents dans la base après filtrage sur origine :</string>
   </property>
  </widget>
  <widget class="Line" name="decorslinebas_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_nbretotal_apres_filtre_targetaudience">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de documents dans la base après filtrage surpublic :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbretotal_filtre_targetaudience">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="decorslinebas_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_titre_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Donner un titre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_titre">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>650</y>
     <width>701</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
